examples/annotations/annotations.bal

```diff
@@ -9,14 +9,26 @@ public function transform(string s) returns string {
     return s.toUpperAscii();
 }
 
-public function main() {
-    // The `@strand` annotation applies to the `start` action.
-    future<int> fut = @strand {thread: "any"} start foo();
+type AnnotRecord record {|
+    string value;
+|};
+
+// Declares an annotation tag on type.
+annotation AnnotRecord annot on type;
 
-    int|error x = wait fut;
-    io:println(x);
+// The `@annot` annotation applies to the `T1` record type.
+@annot {
+    value: "T1"
 }
+type T1 record {
+    string name;
+};
+
+public function main() {
+    T1 a = {name: "John"};
 
-public function foo() returns int {
-    return 10;
+    typedesc<any> t = typeof a;
+    // Access annotation.
+    AnnotRecord? ann = t.@annot;
```

Comment:
@SasinduDilshara believed we might need a new line here for the master branch. I am verifying it.

Reply:
I added a new line assuming this comments comes down in talk bubble in old format